src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleFieldSymbol.vb

```diff
@@ -123,83 +110,58 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+            Locations = If((location Is Nothing), ImmutableArray(Of Location).Empty, ImmutableArray.Create(Of Location)(location))
+            Me.IsImplicitlyDeclared = isImplicitlyDeclared
 
-            Debug.Assert(correspondingDefaultFieldOpt Is Nothing = Me.IsDefaultTupleElement)
+            Debug.Assert(correspondingDefaultFieldOpt Is Nothing = IsDefaultTupleElement)
             Debug.Assert(correspondingDefaultFieldOpt Is Nothing OrElse correspondingDefaultFieldOpt.IsDefaultTupleElement)
 
-            _correspondingDefaultField = If(correspondingDefaultFieldOpt, Me)
+            Me.CorrespondingTupleField = If(correspondingDefaultFieldOpt, Me)
         End Sub
 
         Public Overrides ReadOnly Property Locations As ImmutableArray(Of Location)
-            Get
-                Return Me._locations
-            End Get
-        End Property
 
         Public Overrides ReadOnly Property DeclaringSyntaxReferences As ImmutableArray(Of SyntaxReference)
             Get
-                Return If(_isImplicitlyDeclared,
-                    ImmutableArray(Of SyntaxReference).Empty,
-                    GetDeclaringSyntaxReferenceHelper(Of VisualBasicSyntaxNode)(_locations))
+                Return If(IsImplicitlyDeclared, ImmutableArray(Of SyntaxReference).Empty,
+                                                 GetDeclaringSyntaxReferenceHelper(Of VisualBasicSyntaxNode)(_Locations))
             End Get
         End Property
 
         Public Overrides ReadOnly Property IsImplicitlyDeclared As Boolean
-            Get
-                Return _isImplicitlyDeclared
-            End Get
-        End Property
 
         Friend Overrides ReadOnly Property TypeLayoutOffset As Integer?
             Get
-                Dim flag As Boolean = Me._underlyingField.ContainingType IsNot Me._containingTuple.TupleUnderlyingType
-                Dim result As Integer?
-                If flag Then
-                    result = Nothing
-                Else
-                    result = MyBase.TypeLayoutOffset
-                End If
-                Return result
+                Dim flag As Boolean = UnderlyingField.ContainingType IsNot _containingTuple.TupleUnderlyingType
+                If flag Then Return Nothing
+                Return MyBase.TypeLayoutOffset
             End Get
         End Property
 
         Public Overrides ReadOnly Property AssociatedSymbol As Symbol
             Get
-                Dim flag As Boolean = Me._underlyingField.ContainingType IsNot Me._containingTuple.TupleUnderlyingType
-                Dim result As Symbol
-                If flag Then
-                    result = Nothing
-                Else
-                    result = MyBase.AssociatedSymbol
-                End If
-                Return result
+                Dim flag As Boolean = UnderlyingField.ContainingType IsNot _containingTuple.TupleUnderlyingType
+                If flag Then Return Nothing
```

Comment:
@jcouv I'd considered that but I felt it would make that line too unruly.
So using `flag` is still consistent with existing code, but reduce some of ceremony.